docs/en/create-new-game-map.md

```diff
@@ -9,13 +9,13 @@ To create a new game map, you need to:
 3. Edit `settings.gradle.kts` to include the new subproject `client/game-<mapId>`.
 4. Add an entry in `game-data/i18n-common.yml` in the following format:
 
-```
-- id: «YourNewMapId»
-  data:
-    EN: «The display name of that map»
-```
+    ```yaml
+    - id: «YourNewMapId»
+    data:
+        EN: «The display name of that map»
+    ```
```

Comment:
I'd prefer the old way without indentions... the language marker can be added, though. I'd expect:

```yaml
- id: «YourNewMapId»
  data:
    EN: «The display name of that map»
```

BTW, the indention after the change is wrong, `id` and `data` should have the same indention.